x-pack/plugins/observability/public/pages/slo_edit/slo_edit.test.tsx

```diff
@@ -108,10 +109,13 @@ const mockKibana = () => {
         get: () => {},
       },
       triggersActionsUi: {
-        getAddRuleFlyout: jest
-          .fn()
-          // eslint-disable-next-line @kbn/i18n/strings_should_be_translated_with_i18n, @kbn/i18n/strings_should_be_translated_with_formatted_message
-          .mockReturnValue(<div data-test-subj="add-rule-flyout">Add Rule Flyout</div>),
+        getAddRuleFlyout: jest.fn().mockReturnValue(
+          <div data-test-subj="add-rule-flyout">
+            {i18n.translate('xpack.observability.mockKibana.div.addRuleFlyoutLabel', {
```

Comment:
@maryam-saeidi  - I can do that, but you will still need to approve it since this change came from `main`. Did you still want me to change it back? or can you do it in your PR?